hazelcast/src/test/java/com/hazelcast/jet/core/TestProcessors.java

```diff
@@ -197,13 +227,21 @@ public Function<Address, ProcessorSupplier> get(@Nonnull List<Address> addresses
         }
 
         @Override
-        public void close(Throwable error) {
-            assertEquals("all PS that have been init should have been closed at this point",
-                    MockPS.initCount.get(), MockPS.closeCount.get());
-            assertTrue("Close called without calling init()", initCalled.get());
-            assertTrue("PMS.close() already called once",
-                    closeCalled.compareAndSet(false, true)
-            );
```

Comment:
These assertions don't have an equivalent in the new code.

Reply:
1 - restored
2 - restored
3 - restored by additional method used in only some tests, in which we use MockPMS and have only one job